metricbeat/module/system/fsstats/fsstats.go

```diff
@@ -0,0 +1,94 @@
+// +build darwin linux openbsd windows
+
+/*
+
+An example event looks as following:
+
+    {
+      "@timestamp": "2016-04-26T19:30:19.475Z",
+      "beat": {
+        "hostname": "ruflin",
+        "name": "ruflin"
+      },
+      "metricset": "filesystem",
+      "module": "system",
+      "rtt": 434,
+      "system-filesystem": {
```

Comment:
Is there any reason to have system in the name of the object? Why not just filesystem? I am thinking once we will have conditions in generic filtering, then the name of the field available in the condition is a bit too long (e.g. system-filesystem.device_name). Also, I think a mixture of "-" and "_" is not a good idea.